### PR TITLE
fix: deprecated use of sdkerrors

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Fixes golangci-lint complains about usage of deprecated function

### CODE REVIEW

**Comment by robert-zaremba** [comment]
I will look at the failing tests tomorrow.

**Review by toteki — APPROVED** [review]
Nice clean PR

File: x/oracle/types/msgs_test.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func TestMsgAggregateExchangeRateVote(t *testing.T) {
 	msgInvalidSalt := "invalid salt length; must be 64"
 	msgInvalidOverflowValue := "out of range; bitLen:"
 	msgInvalidHexString := "salt must be a valid hex string: invalid salt format"
-	msgInvalidUnknownRequest := "must provide at least one oracle exchange rate: unknown request"
+	msgInvalidUnknownRequest := "must provide at least one oracle exchange rate: invalid request"
```
Comment (toteki) [inline x/oracle/types/msgs_test.go:94]: This was the only test issue

Comment (gsk967) [reply x/oracle/types/msgs_test.go:94]: I think Yes, It is changed from `Unkown Request` to `Invalid Request` https://github.com/umee-network/umee/pull/1788/files/7909e6d8d5a98eb5763a92b2a30b4903da69e776#diff-0947889d44c2e808b3b719383dd20fd8282c8152705b41c27763555593b82029L114-R114

**Comment by toteki** [comment]
@mergifyio backport release/4.1.x

**Comment by toteki** [comment]
@mergifyio backport release/v4.1.x

**Pull request merged**